app/services/france_travail_api/retrieve_user_token.rb

```diff
@@ -0,0 +1,61 @@
+module FranceTravailApi
+  class RetrieveUserToken < BaseService
+    class UserNotFound < StandardError; end
+    # https://francetravail.io/produits-partages/catalogue/rechercher-usager-v2/documentation#/api-reference/
+
+    def initialize(user:, access_token:)
+      @user = user
+      @access_token = access_token
+    end
+
+    def call
+      send_request!
+      result.user_token = @france_travail_user_token
+    end
+
+    private
+
+    def send_request!
+      response = FranceTravailClient.retrieve_user_token(payload: user_payload, headers: headers)
+      @response_body = JSON.parse(response.body)
+
+      if response.success? && !user_not_found?(response)
```

Comment:
Je crois qu'on peut faire ça non ?
```suggestion
      if response.success? 
```

Reply:
Malheureusement non car comme je l'explique plus bas dans le code, le retour d'erreur "Aucun approchant n'a été trouvé" renvoi une 200 et donc un success... Ca sera possible quand FT aura fix ca (ils sont au courant et doivent faire les changements pour que ce soit une 404).

Reply:
Ah oui my bad 🙏